controller/src/main/java/org/apache/rocketmq/controller/ControllerManager.java

```diff
@@ -91,6 +91,12 @@ protected <T> RunnableFuture<T> newTaskFor(final Runnable runnable, final T valu
             }
         };
         this.heartbeatManager = new DefaultBrokerHeartbeatManager(this.controllerConfig);
+        if (StringUtils.isEmpty(this.controllerConfig.getControllerDLegerPeers())) {
+            throw new IllegalArgumentException("Attribute value controllerDLegerPeers of ControllerConfig is null");
+        }
+        if(StringUtils.isEmpty(this.controllerConfig.getControllerDLegerSelfId())){
+            throw new IllegalArgumentException("Attribute value controllerDLegerSelfId of ControllerConfig is null");
```

Comment:
'is null or empty' may be more accurate.

Reply:
> 'is null or empty' may be more accurate.

OK I will modify it